README.md

```diff
@@ -197,59 +197,83 @@ hits.
 The visual tests use jest-image-snapshot to compare screenshots of the components to their expected state. This
 setup is configured for an exact visual match, so the tests must be run on a consistent platform. The browser used for
 the tests is downloaded via the puppeteer npm package. However, it is also important to run these tests on a consistent
-operating system. The cleanest way to do this is via docker - specifically, to run the tests in a docker environment
-identical to the one which the CI system uses. This environment is defined in the `Dockerfile` present within the
-the top level directory of the repository. To build the docker image, run the following command from this directory:
+operating system. The cleanest way to do this is via Docker - specifically, to run the tests in a Docker environment
+identical to the one which the CI system uses.
+
+To test in a Docker environment, first [install Docker on your computer](https://docs.docker.com/get-docker/). Then,
+within the top level directory of the repository, log into Docker by running the following command. You may be asked to
+input credentials:
 
 ```
-docker build -t rsc-visualtesting .
+docker login docker-all.repo.sonatype.com
 ```
 
-This will create the image and give it a name of "rsc-visualtesting". Note that before running this command, you must
-have docker set up with login credentials for docker-all.repo.sonatype.com.  For non-Sonatype employees, you can instead
-edit the dockerfile to point to the public `node:12` base image instead of the copy hosted on Sonatype's infrastructure.
+Then build a Docker image. You build an image so that you can run tests within a container (environment) based on that
```

Comment:
I added a new sentence here:
> You build an image so that you can run tests within a container (environment) based on that image.

Please let me know if this sentence makes sense.

Reply:
> I added a new sentence here:
> 
> > You build an image so that you can run tests within a container (environment) based on that image.
> 
> Please let me know if this sentence makes sense.

I'm going to assume that this sentence if fine since there was no comment on it.